sklearn/tests/test_multioutput.py

```diff
@@ -601,3 +601,25 @@ def fit(self, X, y, **fit_params):
 
     for est in model.estimators_:
         assert est.sample_weight_ is weight
+
+
+@pytest.mark.parametrize("order_type, order", [
+    [list, [1, 0]],
+    [np.array, [1, 0]],
+    [tuple, [1, 0]],
+    [tuple, [1, 2]]])
+def test_classifier_chain_tuple_order(order_type, order):
+    X = [[1, 2, 3], [4, 5, 6], [1.5, 2.5, 3.5]]
+    y = [[3, 2], [2, 3], [3, 2]]
+    order = order_type(order)
+
+    chain = ClassifierChain(RandomForestClassifier(), order=order)
+
+    if list(order) == [1, 0]:
+        chain.fit(X, y)
+        X_test = [[1.5, 2.5, 3.5]]
+        y_test = [[3, 2]]
+        assert_array_almost_equal(chain.predict(X_test), y_test)
+    else:
+        msg = 'invalid order'
+        assert_raise_message(ValueError, msg, chain.fit, X, y)
```

Comment:
Let's define another test function to test for this assertion and use `pytest.raises`.